libcxx/test/std/containers/views/mdspan/mdspan/conversion.verify.cpp

```diff
@@ -43,6 +43,7 @@ void cant_construct_data_handle_type() {
   int data;
   std::mdspan<int, std::extents<int>, std::layout_right, convertible_accessor_but_not_handle<int>> m_nc(&data);
   // expected-error-re@*:* {{{{.*}}no matching constructor for initialization of {{.*}} (aka 'not_const_convertible_handle<const int>')}}
+  // expected-error@*:* 0-1{{no matching constructor for initialization of 'not_const_convertible_handle<const int>'}}
```

Comment:
Urg. Thanks for fixing these tests. They appear to be way to brittle to changes in compiler diagnostics.